### ISSUE TITLE
IGNORE_FILES doesn't ignore folder in theme ?

### ISSUE THREAD

**Issue by haoadoresorange** [getpelican/pelican#2979]
- [x ] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

## Issue

I have a project structured like this
```
.
├── content
├── Makefile
├── pelicanconf.py
├── publishconf.py
├── tasks.py
├── theme
│   ├── static
│   │   ├── css
│   │   │   └── all.css
│   │   ├── images
│   │   └── sass
│   │       └── all.scss
│   └── templates
```
I wanna ignore the `sass` folder in `theme` folder when generating the output, but I can't make that work with `IGNORE_FILES`. I tried different path for it, relative & absolute variation.
```
IGNORE_FILES = ['sass']
IGNORE_FILES = ['**/sass']
IGNORE_FILES = ['*/**/sass']
```
Only using `IGNORE_FILES = ['*.scss']` to ignore `.scss` files work. But there's still the empty `sass` folder. How do I ignore the folder itself ?

**Comment by justinmayer** [comment]
Hi @haoadoresorange. When filing an issue, please follow the guidelines: https://docs.getpelican.com/en/latest/contribute.html#how-to-get-help

For example, you haven't even included specific `IGNORE_FILES` settings that you have tried. Without doing that, how can you realistically expect someone to help?

**Comment by haoadoresorange** [comment]
> Hi @haoadoresorange. When filing an issue, please follow the guidelines: [docs.getpelican.com/en/latest/contribute.html#how-to-get-help](https://docs.getpelican.com/en/latest/contribute.html#how-to-get-help)
> 
> For example, you haven't even included specific `IGNORE_FILES` settings that you have tried. Without doing that, how can you realistically expect someone to help?

Hi sorry for unclear information, I updated the question above.

**Comment by justinmayer** [comment]
Thanks for the additional detail. Looking through the code, it seems like that setting should behave as you expect.

Indeed, when I tested it myself, everything behaved as expected. My setting:

```
IGNORE_FILES = [
    '*.orig',
    'unused',
]
```

I put several files ending in `[…].css.orig` inside my theme's `static/css/` folder, and I put a folder called `unused` alongside the `css`, `fonts`, `js`, and other folders inside my theme's `static/` folder, and indeed neither the `*.orig` files nor the `unused` folder were found in the generated output.

In short, I don't know why you are not seeing the same behavior. You could try running Pelican in debug mode, but I'm not sure whether that would yield useful clues as to the cause of the behavior you are seeing.

**Comment by haoadoresorange** [comment]
Indeed, I couldn't reproduce it anymore. I made a couple changes to the settings since this post. It might be something that interfere at first that got removed now. So I'm closing the issue here.

**Comment by justinmayer** [comment]
Glad to hear everything is working well now. ✨
